code/modules/events/_event.dm

```diff
@@ -132,11 +132,15 @@ Runs the event
 		log_game("Random Event triggering: [name] ([typepath]).")
 
 	if(alert_observers)
-		deadchat_broadcast(" has just been[random ? " randomly" : ""] triggered!", "<b>[name]</b>", message_type=DEADCHAT_ANNOUNCEMENT) //STOP ASSUMING IT'S BADMINS!
+		annouce_name(random)
 
 	SSblackbox.record_feedback("tally", "event_ran", 1, "[E]")
 	return E
 
+///Used for the beer nuke/custom scrubber overflow which broadcast a different name to their actual event name
+/datum/round_event_control/proc/annouce_name(random)
```

Comment:
Please give this another name, like /proc/announce_deadchat
Also make the comment more generic about what it actually does please